src/app/pages/card-import/card-import.page.ts

```diff
@@ -37,168 +34,181 @@ export class CardImportPage implements OnInit {
     targetHeight: 374
   }
 
-  constructor(private route: ActivatedRoute, private router: Router, private camera: Camera, private toastCtrl: ToastController, public alertController: AlertController, private cardService: CardService) {
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router,
+    private camera: Camera,
+    private toastCtrl: ToastController,
+    public alertController: AlertController,
+    public authService: AuthService,
+    private cardService: CardService
+  ) {
+    console.log('Card Import Start');
+  }
+
+  ngOnInit() {
+    console.log('Card Import Init');
+
+    this.id = this.route.snapshot.paramMap.get('id');
 
+    if (this.id) {
+      console.log('Current card ID: ', this.id);
+      this.cardService.getCard(this.id).subscribe(card => {
+        console.log("Card: ", card);
+        this.card = card;
+        this.frontImg = this.card.frontImg;
+        this.backImg = this.card.backImg;
+      });
+    } else {
+      console.log('New card');
+      this.importCardFront();
+    }
   }
 
-   ngOnInit() {
```

Comment:
Yo! Thanks for tidying up behind me! Improving readability will be very key as more hands are called on deck.

Reply:
I can't abide by an odd number of spaces. In some languages, instead of curly braces and semicolons, they use the actual indentation to determine nesting. It's a good practice to develop.